packages/space-opera/src/components/model_viewer_snippet/components/open_button.ts

```diff
@@ -157,7 +157,8 @@ export class OpenModal extends LitElement {
             <textarea id="mv-input" rows=10>${exampleLoadableSnippet}</textarea>
             ${this.errors.map(error => html`<div>${error}</div>`)}
           </div>
-          <div>
+          <div class="mv-note">Edit the top snippet. Then update the bottom snippet, which sets the &lt;model-viewer&gt paramters in the editor.</div>
+          <div class="update-button">
```

Comment:
I think this two-snippet thing is unnecessary: The done button should do what the update button does. Might be nice to have a cancel button too. Actually, I think this modal thing should just be for snippet input, and put a top-level button in for GLB import; that'll be more symmetric with the GLB and ZIP export buttons below.

Reply:
Also, it's a bit confusing that loading a "default model" is done by updating the text snippet. We already had a feature request to have a dropdown to open a few provided models like the old interactive example did. It would be great if we could put that feature in so that the snippet didn't need to provide that functionality.

Reply:
yep, done

Reply:
new modal: https://screenshot.googleplex.com/6pXzkVAPbg3hPdR.png

Reply:
new file manager: https://screenshot.googleplex.com/6qx8SgxxmQBoSGq.png

Reply:
Those are looking good! Is it possible to have the buttons on their own instead of inside a named box? I feel like the box and name aren't adding much but extra space.

Reply:
It would still need some kind of descriptor next to it anyway. I think given what we are asking it to do: Be a quickly recognizable feature, I think it's okay.